docs/machine-learning/how-to-guides/train-machine-learning-model-ml-net.md

```diff
@@ -0,0 +1,179 @@
+---
+title: "How-To: Train a machine learning model in ML.NET" 
+description: Learn how to train various models in ML.NET
+ms.date: 04/09/2019
+author: luisquintanilla
+ms.author: luquinta
+ms.custom: mvc,how-to
+#Customer intent: As a developer I want to know how to train my model
+---
+
+# How-To: Learn how to train a machine learning model in ML.NET
+
+This how-to shows you how to build machine learning models, extract learned parameters and measure performance with ML.NET. Although this sample goes through the training of a regression model, the concepts are applicable throughout a majority of the other algorithms.
+
+## Split data for training and testing
+
+The goal of a good machine learning model is to generalize well or predict unseen observations with more accuracy than those it was trained on. To do so, it is recommended to split data into training and test sets. 
```

Comment:
those = the initial observations?  "it is" -> "it's" Could we be more prescriptive and use something like:  "To do so, split data into training and test sets. "?